internal/datacoord/compaction_task_clustering.go

```diff
@@ -116,7 +116,7 @@ func (t *clusteringCompactionTask) Process() bool {
 		if t.GetTaskProto().State == datapb.CompactionTaskState_completed || t.GetTaskProto().State == datapb.CompactionTaskState_cleaned {
 			updateOps = append(updateOps, setEndTime(ts))
 			elapse := ts - t.GetTaskProto().StartTime
-			log.Info("clustering compaction task total elapse", zap.Int64("elapse seconds", elapse))
+			log.Info("clustering compaction task total elapse", zap.Duration("costs", time.Duration(elapse)*time.Second))
```

Comment:
no need to multiply the time.Second

Reply:
Since the start time is in Unix format, the elapsed time needs to be multiplied by time.Second. refer to 
https://go.dev/play/p/U2A8QtEpYdX